fda-model/alternatives/Alternative.cs

```diff
@@ -145,34 +140,33 @@ public AlternativeResults AnnualizationCompute(interfaces.IProvideRandomNumbers
                     {
                         double eadSampledBaseYear = baseYearDamageResult.ConsequenceHistogram.InverseCDF(randomProvider.NextRandom());
                         double eadSampledFutureYear = mlfYearDamageResult.ConsequenceHistogram.InverseCDF(randomProvider.NextRandom());
-                        double aaeqDamage = ComputeEEAD(eadSampledBaseYear, eadSampledFutureYear);
+                        double aaeqDamage = ComputeEEAD(eadSampledBaseYear, baseYear, eadSampledFutureYear, futureYear, periodOfAnalysis, discountRate);
                         aaeqResult.AddConsequenceRealization(aaeqDamage, i);
                     }
                     aaeqResult.ConsequenceHistogram.ForceDeQueue();
-                    aaeqResults.AddConsequenceResult(aaeqResult);
+                    alternativeResults.AddConsequenceResults(aaeqResult);
                 }
-                alternativeResults.AddConsequenceResults(aaeqResults);
             }
             return alternativeResults;
         }
         //TODO: these functions should be private, but currently have unit tests 
         //so these will remain public until the unit tests are re-written on the above public method
-        public double ComputeEEAD(double baseYearEAD, double mostLikelyFutureEAD){
+        public static double ComputeEEAD(double baseYearEAD, int baseYear, double mostLikelyFutureEAD, int mostLikelyFutureYear, int periodOfAnalysis, double discountRate){
 
             //probably instantiate a rng to seed each impact area differently
```

Comment:
The word "probably" makes it sound like we are not sure what to do here. Lets either change the message or add a "TODO:" in front of it to evaluate later. This way we can search all of FDA for the words "todo" and try to clean everything up at a later date.